File: Assignment 6/Activity 1.js
// This program calculates  their weekly, monthly, and annual gross pay.
//
// References:
//  https://en.wikibooks.org/wiki/JavaScript

function main() {
    
    let hours;
    let rateperhour;
    let weekly;
    let monthly;
    let annual;
    
    hours = gethours();
    rateperhour = getrateperhour();
    weekly = calculateweekly(rateperhour, hours);
    monthly = calculatemonthly(rateperhour, hours);
    annual = calculateannual(rateperhour, hours);
    displayResult(weekly, monthly, annual);
}

function calculatemonthly(hours, rateperhour) {
    let monthly;
    
    monthly = hours * rateperhour * 4;
    
    return monthly;
}

function calculateannual(hours, rateperhour) {
    let annual;
    
    annual = hours * rateperhour * 52;
    
    return annual;
}

function calculateweekly(hours, rateperhour) {
    let weekly;
    
    weekly = hours * rateperhour;
    
    return weekly;
}

function displayResult(weekly, monthly, annual) {
    window.alert(weekly.toString() + " weekly " + monthly + " monthly " + annual + " annually ");
}

function gethours() {
    let hours;
    
    window.alert("Enter hours worked this week");
    hours = window.prompt('Enter a value for hours');
    
    return hours;
}

function getrateperhour() {
    let rateperhour;
    
    window.alert("Enter rate per hour");
    rateperhour = window.prompt('Enter a value for rateperhour');
    
    return rateperhour;
}

File: Assignment 6/Activity 1.py
# This program calculates their weekly, monthly, and annual gross pay.
# References: https://en.wikibooks.org/wiki/JavaScript

def calculate_monthly(hours, rate_per_hour):
    monthly = hours * rate_per_hour * 4
    return monthly


def calculate_annual(hours, rate_per_hour):
    annual = hours * rate_per_hour * 52
    return annual


def calculate_weekly(hours, rate_per_hour):
    weekly = hours * rate_per_hour
    return weekly


def display_result(weekly, monthly, annual):
    print(str(weekly) + " weekly " +
    str(monthly) + " monthly " +
    str(annual) + " annually ")


def get_hours():
    print("Enter hours worked this week")
    hours = float(input())
    return hours


def get_rate_per_hour():
    print("Enter rate per hour")
    rate_per_hour = float(input())
    return rate_per_hour

# main function starts here

hours = get_hours()
rate_per_hour = get_rate_per_hour()
weekly = calculate_weekly(rate_per_hour, hours)
monthly = calculate_monthly(rate_per_hour, hours)
annual = calculate_annual(rate_per_hour, hours)
display_result(weekly, monthly, annual)
